terralab/commands/pipeline_runs_commands.py

```diff
@@ -47,9 +60,88 @@ def submit(pipeline_name: str, version: int, inputs: str, description: str):
 )
 @handle_api_exceptions
 def download(pipeline_name: str, job_id: str, local_destination: str):
-    """Download all output files from a PIPELINE_NAME pipeline run with JOB_ID identifier"""
+    """Download all output files from a PIPELINE_NAME job with JOB_ID identifier"""
     job_id_uuid: uuid.UUID = validate_job_id(job_id)
 
     pipeline_runs_logic.get_result_and_download_pipeline_run_outputs(
         pipeline_name, job_id_uuid, local_destination
     )
+
+
+@click.command(short_help="Get the status and details of a job")
+@click.argument("pipeline_name", type=str)
+@click.argument("job_id", type=str)
+@handle_api_exceptions
+def details(pipeline_name: str, job_id: str):
+    """Get the status and details of a PIPELINE_NAME job with JOB_ID identifier"""
+    job_id_uuid: uuid.UUID = validate_job_id(job_id)
+
+    response: AsyncPipelineRunResponse = pipeline_runs_logic.get_pipeline_run_status(
+        pipeline_name, job_id_uuid
+    )
+
+    LOGGER.info(
+        add_blankline_after(f"Status: {format_status(response.job_report.status)}")
+    )
+
+    if response.error_report:
+        LOGGER.info(
+            add_blankline_after(f"Error message: {response.error_report.message}")
+        )
+
+    LOGGER.info("Details:")
+    col_size = 19
```

Comment:
because (imo) the size of the column should be just wider than the longest text in the first column. i don't think we use this consistently enough across calls to make it worth trying to have it be consistent spatially everywhere. thoughts?

Reply:
ahhh i see, so how do we make sure if we change some text the col_size isnt bad.  should the `pad_column` function make sure the padding is larger than the text its trying to pad?

Reply:
hmm, I think I have a higher tolerance for having to manually check things than you do. I feel like I'd want that behavior (pad_column throws an error if the padding is smaller than the text) to happen in our tests but not in the actual running of the CLI. as long as the fields being padded are not determined by user input, that would happen if we include that behavior, but I could imagine a future in which the padded text is not predictable and I wouldn't want it throwing an error for the user in that case. let's chat about this

Reply:
we agreed to have the pad_column function add a space if the requested padding is too small, otherwise remain the same

Reply:
thank you for teh solution we discussed, love it!